app/src/main/java/br/org/cesar/knot_setup_app/fragment/configureGatewayWifi/ConfigureGatewayWifiFragment.java

```diff
@@ -0,0 +1,79 @@
+package br.org.cesar.knot_setup_app.fragment.configureGatewayWifi;
+
+import android.os.Bundle;
+import android.support.v4.app.Fragment;
+import android.text.TextUtils;
+import android.util.Log;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.Button;
+import android.widget.EditText;
+import android.widget.Toast;
+
+import br.org.cesar.knot_setup_app.R;
+import br.org.cesar.knot_setup_app.fragment.configureGatewayWifi.ConfigureGatewayWifiContract.ViewModel;
+import br.org.cesar.knot_setup_app.fragment.configureGatewayWifi.ConfigureGatewayWifiContract.Presenter;
+import br.org.cesar.knot_setup_app.wrapper.LogWrapper;
+
+public class ConfigureGatewayWifiFragment extends Fragment implements ViewModel {
+
+    private EditText wifiEditText;
```

Comment:
In the previous commit I think you used `somethingFld` (which should be replaced by Field). But try to keep a standard.

Reply:
I decided to adopt this pattern <variable name><XmlField> in order to keep a clear distinction from text fields to edit texts, I would like to hear your thoughts about this. Also, are you requesting any changes? I couldn't tell from the comment.